config/core/roles/webhook-clusterrole.yaml

```diff
@@ -128,6 +128,15 @@ rules:
       - "create"
       - "patch"
 
+  - apiGroups:
+      - eventing.knative.dev
+    resources:
+      - eventpolicies
+    verbs:
+      - get
+      - list
+      - watch
```

Comment:
This can be a bit confusing. The webhook is not directly using the EventPolicyLister. But the SinkBinding uses the OIDCTokenProvider from the auth package. In this package we also have the OIDCTokenVerifier, which uses the EventPolicyLister (via eventpolicylister.Get()) and thus registers the informer in its init() method